ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -117,7 +122,10 @@ def main(argv=sys.argv[1:]):
             'severity': error.get('severity'),
             'msg': error.get('verbose'),
         }
-        report[filename].append(data)
+
+        file_report = report.get(filename, [])
+        file_report.append(data)
+        report.setdefault(filename, file_report)
```

Comment:
Instead of calling `get` and `setdefault` this can be simplified to:

```
file_report = report.setdefault(filename, [])
file_report.append(data)
```

As an alternative the type of `report` could be changed to `defaultdict(list)`. That would avoid the extract call altogether.

Reply:
Done.